aiounifi/models/device.py

```diff
@@ -722,6 +723,43 @@ def create(
         )
 
 
+@dataclass
+class DeviceSetLedStatus(ApiRequest):
+    """Request object for setting LED status of device."""
+
+    @classmethod
+    def create(
+        cls,
+        device: "Device",
+        status: str = "on",
+        brightness: int | None = None,
+        color: str | None = None,
+    ) -> Self:
+        """Set LED status of device."""
+
+        data = {"led_override": status}
+        if device.supports_led_ring:
+            # Validate brightness parameter
+            if brightness is not None and (0 <= brightness <= 100):
+                data["led_override_color_brightness"] = brightness
+            else:
+                raise AttributeError("Brightness must be within the range [0, 100].")
+
+            # Validate color parameter
+            if color is not None and re.match(r"^#(?:[0-9a-fA-F]{3}){1,2}$", color):
+                data["led_override_color"] = color
+            else:
+                raise AttributeError(
+                    "Color must be a valid hex color code (e.g., '#00FF00')."
+                )
```

Comment:
Is it required it send both brightness and led_override_color?

```suggestion
        if device.supports_led_ring and brightness is not None and color is not None:
            # Validate brightness parameter
            if 0 <= brightness <= 100:
                data["led_override_color_brightness"] = brightness
            else:
                raise AttributeError("Brightness must be within the range [0, 100].")

            # Validate color parameter
            if re.match(r"^#(?:[0-9a-fA-F]{3}){1,2}$", color):
                data["led_override_color"] = color
            else:
                raise AttributeError(
                    "Color must be a valid hex color code (e.g., '#00FF00')."
                )
```

Reply:
I don't believe it's required to send both together. Can double check against my environment, but I don't believe you need to send both.